peekingduck/pipeline/nodes/input/recorded.py

```diff
@@ -53,9 +55,19 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         '''
         outputs = self._run_single_file()
 
+        _processing = round((self.frame_counter/self.videocap.frame_count)*100)
```

Comment:
We usually have an underscore in front for private functions, but not for variables.